### ISSUE TITLE
clarification on setting session using token

### ISSUE THREAD

**Issue by johndpope** [supabase-community/auth-py#221]
related 
https://github.com/supabase-community/supabase-swift/issues/49

**Comment by silentworks** [comment]
Closing this out as there has been no activity on this issue and I think this is no longer an issue.

**Comment by johndpope** [comment]
all good with this helper now thanks - 
https://github.com/supabase-community/supabase-py/issues/616

**Comment by luquitared** [comment]
curious - when setting a session with access_token, do RLS policies apply?

**Comment by johndpope** [comment]
naturally / logic is all server side. there's libraries that can talk with postgres directly that maybe suitable for bypassing RLS.

**Comment by luquitared** [comment]
When I first left my comment here on Nov 19, 2022, I dropped using supabase because the python client did not respect RLS.

I write a lot of JS frontend, which has outstanding support for supabase (the library is fantastic) but never solved this RLS issue and never revisited to see if it was solved.

Basically, if user passes their JWT to my python server, I wanted the RLS policies to apply.

I had the same issue as this person:
https://github.com/supabase-community/supabase-py/issues/616
